integration-test/src/main/java/com/sequenceiq/it/cloudbreak/PrivateEndpointTest.java

```diff
@@ -0,0 +1,13 @@
+package com.sequenceiq.it.cloudbreak;
+
+public class PrivateEndpointTest {
```

Comment:
Usually we `extends GherkinTest` at this level of the project. If you do not judge, this is not worth it, we can leave this at it is.

Reply:
I see. What is the benefit of extending the GherkinTest class? I see that the methods of the GherkinTest class are used solely in RepositoryConfigTests.

Reply:
Ok, you convinced me via chat, I change it. It helps the organization.